SQLHelper/pom.xml

```diff
@@ -32,4 +32,28 @@
     <maven.compiler.source>15</maven.compiler.source>
     <maven.compiler.target>15</maven.compiler.target>
   </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <version>2.4</version>
+        <executions>
+          <execution>
+            <id>make-zip</id>
+            <phase>package</phase>
+            <goals>
+              <goal>single</goal>
+            </goals>
+            <configuration>
+              <descriptors>
+                <descriptor>./assembly.xml</descriptor>
+              </descriptors>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
You could take a look on here: http://maven.apache.org/plugins/maven-assembly-plugin/usage.html, chapter "Execution: Building an Assembly". 

maven-assembly-plugin provides a pre-defined descriptor called "jar-with-dependencies", I think we could use it instead of creating another `assembly.xml`

Reply:
Please also upgrade the version of maven-assembly-plugin